core/src/main/java/com/statful/client/core/StatfulClientImpl.java

```diff
@@ -93,6 +93,17 @@ public final SenderFacade gauge(final String metricName, final Integer value) {
     }
 
     @Override
+    public final SenderFacade metric(final String metricName, final Integer value) {
+        SenderAPI senderAPI = MetricsSenderAPI.newInstance(this).with()
+                .configuration(configuration)
+                .aggFreq(configuration.getDefaultAggregationFreq())
```

Comment:
What's the logic here? We don't know the type of the metric, how can we infer a default aggregation?